package.json

```diff
@@ -9,14 +9,19 @@
   "homepage": "https://github.com/cabal-club/cabal-client",
   "bugs": "https://github.com/cabal-club/cabal-client/issues",
   "main": "index.js",
+  "browser": {
+    "mkdirp": false,
+    "random-access-file": "random-access-web"
+  },
   "scripts": {
     "test": "tape test/*.js",
     "lint": "standard",
-    "doc": "jsdoc2md --files src/client.js src/cabal-details.js --param-list-format list --separators --partial doc-gen/scope.hbs --helper doc-gen/helpers.js > api.md"
+    "doc": "jsdoc2md --files src/client.js src/cabal-details.js --param-list-format list --separators --partial doc-gen/scope.hbs --helper doc-gen/helpers.js > api.md",
+    "build": "browserify boop.js > bundle.js"
```

Comment:
Did you omit boop.js? :D

Also i found that i needed to specify `standalone` when browserifying so that i could get access to it from my external code... like this:

browserify command:
```browserify index.js --standalone CabalClient > bundle.js```

and then to use it like:
```
    <script src="bundle.js"></script>
    <script>
      var client = new CabalClient({
       ...
```

at least that's what seemed to work. but what do i know?! although i presume i'll need to access it in the same way when pulling it in in react native

Reply:
lol yeah i think that's a mistake; boop.js might've just been me testing things ^^